src/main/java/io/vertx/core/impl/ContextImpl.java

```diff
@@ -228,10 +232,6 @@ public EventLoop getEventLoop() {
       });
     } catch (RejectedExecutionException ignore) {
       // Pool is already shut down
-    } finally {
-      if (blockingCodeHandler != null) {
```

Comment:
that was leaving null thread contextclassloader of the junit thread and issues in subsequents jvm-npm.js deployment